src/modules/avformat/consumer_avformat.c

```diff
@@ -354,63 +356,12 @@ static void color_trc_from_colorspace(mlt_properties properties)
 static void color_primaries_from_colorspace(mlt_properties properties)
 {
     // Default color_primaries from MLT colorspace.
-    switch (mlt_properties_get_int(properties, "colorspace")) {
-    case 0: // sRGB
-    case 709:
-        mlt_properties_set_int(properties, "color_primaries", AVCOL_PRI_BT709);
```

Comment:
This is one spot I am unsure of. Why were we setting the properties with FFMpeg values? Everywhere else we were setting the MLT values (709, 601525, 601625). Was this a bug? Or is there a reason we were setting "color_primaries" to mlt values in some places and FFMpeg values in this one place?